test/unit_tests/extension.test.ts

```diff
@@ -710,11 +700,10 @@ suite('Diagnostics', async () => {
   });
 });
 
-import dirs from '../src/dirs';
 
 suite('Compilation info', () => {
   test('Parsing compilation databases', async () => {
-    const dbpath = testFilePath('test_compdb.json');
+    const dbpath = getTestRessourceFilePath('test_compdb.json');
```

Comment:
Should be spelled `getTestResourceFilePath`

Reply:
Done